Xamarin.Forms.Platform.iOS/Renderers/ShellSectionRenderer.cs

```diff
@@ -174,17 +167,12 @@ protected virtual void HandlePropertyChanged(object sender, PropertyChangedEvent
 				UpdateTabBarItem();
 		}
 
-		protected virtual IShellSectionRootRenderer CreateShellSectionRootRenderer(ShellSection shellSection, IShellContext shellContext)
```

Comment:
Can you bring this back?

Reply:
Done! The only parts that are important are the removal of the `PopViewController` and the changes to the `NavDelegate`. My mistake, was copying things around... should've paid more attention to the diff!